src/components/Combobox/__tests__/__snapshots__/combobox-snapshot-tests.jest.js.snap

```diff
@@ -946,7 +946,9 @@ exports[`Combobox renders correctly with loading 1`] = `
                 title="loading"
               >
                 <svg
+                  aria-hidden={true}
                   className="circle-loader-spinner circleLoaderSpinner input-component__loader-svg"
+                  role="presentation"
```

Comment:
it's weird for me that we needed both aria-hidden={true} and  role="presentation" but I saw talya wrote it in the review. checking.

Reply:
Talked with Talya, let's keep only aria-hidden={true}.
I already approve the pr for not block you from moving forward but let's fix it before merging.

Reply:
Done